pkg/broker/broker.go

```diff
@@ -1398,6 +1398,7 @@ func (a AnsibleBroker) validateRequestedUpdateParams(
 			// can make sense of them.
 			log.Noticef("Requested update parameter: [%v] with the value of [%v] was found " +
 				"in an update request, but it did not exist on the previous service instance.")
+			changedParams[reqParam] = reqVal
```

Comment:
If this is all we are going to do then we should just remove the entire if/else chain and always update IMO

Reply:
I'm not against that.